js_modules/dagster-ui/packages/ui-core/src/assets/AllIndividualEventsButton.tsx

```diff
@@ -293,7 +294,11 @@ const DetailsTable = styled.table`
 interface PredecessorDialogProps {
   hasLineage: boolean;
   hasPartitions: boolean;
-  events: (AssetMaterializationFragment | AssetObservationFragment)[];
+  events: (
+    | AssetSuccessfulMaterializationFragment
+    | AssetObservationFragment
+    | AssetFailedToMaterializeFragment
```

Comment:
I don't suppose there's a FailedToObserveFragment too? 😅

Reply:
There is not. I inquired about this here https://dagsterlabs.slack.com/archives/C06FEEBPQJF/p1742332232465729